app/scripts/components/common/map/map-wrapper.tsx

```diff
@@ -0,0 +1,116 @@
+import React, {
+  ReactNode,
+  Children,
+  useMemo,
+  ReactElement,
+  JSXElementConstructor,
+  useState,
+  createContext,
+} from 'react';
+import styled from 'styled-components';
+import useDimensions from 'react-cool-dimensions';
+import { useMap } from 'react-map-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import 'mapbox-gl-compare/dist/mapbox-gl-compare.css';
+import MapboxStyleOverride from './mapbox-style-override';
+import { Styles } from './styles';
+import useMapCompare from './hooks/useMapCompare';
+import MapComponent from './map-component';
+
+
+const MapWrapperContainer = styled.div`
+  && {
+    inset: 0;
+    height: 100%;
+  }
+
+  & > * {
+    position: absolute !important;
+    top: 0;
+    bottom: 0;
+    left: 0;
+  }
+
+  ${MapboxStyleOverride}
```

Comment:
```suggestion
  ${MapboxStyleOverride}
  
  height: 100%;
```

The map container itself should not be absolutely positioned. It is better if the parent takes care of positioning it.

The parent, being a flex container will ensure that the map takes up the full size with just `height: 100%`.

Now, the mock controls and the compare button also take up space, but they should be moved elsewhere as they do not belong there.

Reply:
> The parent, being a flex container will ensure that the map takes up the full size with just height: 100%.

It should, yes, but here `MapsContainer` while being set at 100% height, doesn't occupy the whole vertical space. Not sure what's going on here :/